admin/RELEASE.md

```diff
@@ -4,6 +4,17 @@
 > Updated for `4.1.2` on May 17, 2021.
 > -MGatner
 
+## Labeling PRs
+
+To auto-generate changelog, each PR to be listed in changelog must have one of the following [labels](https://github.com/codeigniter4/CodeIgniter4/labels):
+- **bug** ... PRs that fix bugs
+- **enhancement** ... PRs to improve existing functionalities
+- **new feature** ... PRs for new features
+- **refactor** ... PRs to refactor
+
+And a PRs that has the breaking changes must have the following label:
+- **breaking change** ... PRs that may break existing functionalities
+
```

Comment:
I like having this here for reference, but it should also be in https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md so submitters (hopefully) apply their own labels.

Reply:
Okay, I will add.

"Apply/dismiss labels" needs `Triage` role.
https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role

Reply:
Added.